kuma/static/js/components/mathml.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * vanilla version of the function at:
+ * https://github.com/mdn/kuma/blob/master/kuma/static/js/wiki.js#L252
+ */
+(function() {
+    'use strict';
+
+    /*
+     * Bug 981409 - Add some CSS fallback for browsers without MathML support.
+     *
+     * This is based on:
+     * https://developer.mozilla.org/en-US/docs/Web/MathML/Authoring#Fallback_for_Browsers_without_MathML_support
+     * and https://github.com/fred-wang/mathml.css
+     */
+    var mathElements = document.querySelectorAll('math');
+    // if no `math` elements on the page, bail
+    if (!mathElements.length) {
+        return;
+    }
+
+    /**
+     * Returns a `math` element wrapped in a `div` that is positioned offscreen
+     * @returns `div` element
+     */
+    function getMathElement() {
+        var offscreenContainer = document.createElement('div');
+        var mathMLNamespace = 'http://www.w3.org/1998/Math/MathML';
+        var mathElement = document.createElementNS(mathMLNamespace, 'math');
+        var mspaceElement = document.createElementNS(mathMLNamespace, 'mspace');
+
+        mspaceElement.setAttribute('height', '23px');
+        mspaceElement.setAttribute('width', '77px');
+
+        mathElement.append(mspaceElement);
+        offscreenContainer.append(mathElement);
+        offscreenContainer.classList.add('offscreen');
+
+        return offscreenContainer;
+    }
+
+    // Test for MathML support
+    var mathMLTestElement = document.body.appendChild(getMathElement());
+    var box = mathMLTestElement.querySelector('mspace').getBoundingClientRect();
```

Comment:
To my point of the earlier comment: this would break all script execution if a page contains a `<math>` element but not a `<mspace>` element. Now I admit that seems unlikely, but it's still scary that an extra element _somewhere_ could break the whole page's JS. I wouldn't make this code defensive though, once it's separated from the main bundle it can just error out.

What I'd do though (in general) is move `querySelector` & co to the top of the block. That way it's at least clear what the template "requirements" are for each script.